docs/source/DeveloperGuide/TornadoServices/index.rst

```diff
@@ -0,0 +1,330 @@
+    }
+  }
+
+
+But you can also control more settings by launching tornado yourself::
+
+  from DIRAC.Core.Tornado.Server.TornadoServer import TornadoServer
+  serverToLaunch = TornadoServer(youroptions)
+  serverToLaunch.startTornado()
+
+Options available are:
+
+- services, should be a list, to start only these services
+- debugSSL, True or False, activate debug mode of Tornado (includes autoreload) and SSL, for extra logs use -ddd in the command line
+- port, int, if you want to override value from config. If it's also not defined in config, it use 443.
+
+This start method can be useful for developing new service or create starting script for a specific service, like the Configuration System (as master).
+
+TransferClient
+**************
+
+There is no specific client for transfering files anymore. In fact, the whole idea of directly serving file will eventually disapear and be replaced with redirections to real content streaming server. In the meantine, in order to keep some compatibility, the features were implemented, but require some changes on the server side:
+
+- ``transfer_toClient`` needs to be renamed ``export_streamToClient``
+- It needs to return the whole file content at once
+- The parameter ``fileHelper`` is removed
+
+
+For example::
+
+  def transfer_toClient(self, myFileToSend, token, fileHelper):
+
+    # Do whatever with the token
+
+    with open(myFileToSend, 'r') as fd:
+      ret = fileHelper.DataSourceToNetwork(fd)
+      return ret
+
+Simply becomes::
+
+  def export_streamToClient(self, myFileToSend, token):
+
+    # Do whatever with the token
+
+    with open(myFileToSend, 'r') as fd:
+      return fd.read()
+
+
+From the client side, no change is needed since :py:meth:`DIRAC.Core.Tornado.Client.TornadoClient.TornadoClient.receiveFile` keeps the interface
+
+This procedure is not optimized server side (see commented ``export_streamToClient`` implementation in :py:class`DIRAC.Core.Tornado.Server.TornadoService.TornadoService`). 
+
+The ``transfer_fromClient`` equivalent has not yet been implemented as it concerns only very few cases (basically DIRAC SE and SandboxStore)
+
+******
+Client
+******
+
+.. graphviz::
+
+   digraph {
+   TornadoClient -> TornadoBaseClient [label=inherit]
+   TornadoBaseClient -> Requests [label=use]
+
+   TornadoClient  [shape=polygon,sides=4, label="DIRAC.Core.Tornado.Client.TornadoClient"];
+   TornadoBaseClient  [shape=polygon,sides=4, label="DIRAC.Core.Tornado.Client.private.TornadoBaseClient"];
+   Requests [shape=polygon,sides=4]
+   }
+
+This diagram present what is behind TornadoClient, but you should use :py:class:`DIRAC.Core.Base.Client` ! The new client integrate a selection system which select for you between HTTPS and DISET client.
+
+In your client module when you inherit from :py:class:`DIRAC.Core.Base.Client` you can define `httpsClient` with another client, it can be useful when you can't serialize some data in JSON. Here the step to create and use a JSON patch:
```

Comment:
```suggestion
In your client module when you inherit from :py:class:`DIRAC.Core.Base.Client` you can define ``httpsClient`` with another client, it can be useful when you can't serialize some data in JSON. Here the step to create and use a JSON patch:
```